README.md

```diff
@@ -7,42 +7,41 @@ Balena CLI
 [![dependencies](https://david-dm.org/balena-io/balena-cli.svg)](https://david-dm.org/balena-io/balena-cli)
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/balena-io/chat)
 
-Requisites
-----------
-
-If you want to install the CLI directly through npm, you'll need the below. If this looks difficult,
-we do now have an experimental standalone binary release available, see ['Standalone install'](#standalone-install) below.
-
-- [NodeJS](https://nodejs.org) (>= v6)
-- [Git](https://git-scm.com)
-- The following executables should be correctly installed in your shell environment:
-  - `ssh`: Any recent version of the OpenSSH ssh client (required by `balena sync` and `balena ssh`)
-	- if you need `ssh` to work behind the proxy you also need [`proxytunnel`](http://proxytunnel.sourceforge.net/) installed (available as `proxytunnel` package for Ubuntu, for example)
-  - `rsync`: >= 2.6.9 (required by `balena sync`)
-
-##### Windows Support
+Getting Started
+---------------
 
-Before installing balena-cli, you'll need a working node-gyp environment. If you don't already have one you'll see native module build errors during installation. To fix this, run `npm install -g --production windows-build-tools` in an administrator console (available as 'Command Prompt (Admin)' when pressing windows+x in Windows 7+).
+The easiest and recommended way of installing the CLI on all platforms (Linux, MacOS, Windows)
+is to use the Standalone Install described below. Some specific CLI commands like `balena ssh`
+and `balena sync` have additional dependencies: see section Additional Dependencies.
 
-`balena sync` and `balena ssh` have not been thoroughly tested on the standard Windows cmd.exe shell. We recommend using bash (or a similar) shell, like Bash for Windows 10 or [Git for Windows](https://git-for-windows.github.io/).
+> **Windows users:** we now have a [YouTube video tutorial](https://www.youtube.com/watch?v=j3JoA1EINUA)
```

Comment:
@dansku, in this commit, subject to review and comments, I have moved the video link here.

Reply:
Looks good to me!